module-utils/log/CMakeLists.txt

```diff
@@ -26,6 +26,25 @@ target_link_libraries(log
         sys-service
 )
 
+if(${PROJECT_TARGET} STREQUAL "TARGET_RT1051")
+    if(${PRODUCT} STREQUAL "PurePhone")
+        set(CIRCULAR_BUFFER_SIZE 1024)
```

Comment:
There can be several circular buffers in the project. I propose to rename it to something like `LOGGER_BUFFER_SIZE` to indicate precisely what it is.